graphql_compiler/tests/integration_tests/integration_test_helpers.py

```diff
@@ -45,8 +45,19 @@ def compile_and_run_match_query(schema, graphql_query, parameters, orientdb_clie
     }
     compilation_result = graphql_to_match(schema, graphql_query, converted_parameters)
 
+    # Get results, adding None for optional columns with no matches
     query = compilation_result.query
-    results = [row.oRecordData for row in orientdb_client.command(query)]
+    results = []
+    for row in orientdb_client.command(query):
+        row_dict = row.oRecordData
+        for output_name, output_info in six.iteritems(compilation_result.output_metadata):
+            if output_name not in row_dict:
+                if not output_info.optional:
+                    raise AssertionError(
```

Comment:
This can get hit if the database has `null` as the value in some column. Is that something we guarantee the test data does not have?

Reply:
Oh OrientDB returns nothing even for null values? In that case I'll take this assertion back.